from django.core.exceptions import ValidationError


def validate_text_signature(value):
    # TODO
    pass
